packages/frontend/next.config.js

```diff
@@ -1,4 +1,8 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
 
-module.exports = nextConfig
+// Allows us to use sockets
+const nextConfig = {
+  transpilePackages: ["bufferutil", "utf-8-validate"],
+};
```

Comment:
I don't really understand why we have to do this, but it stopped the weird messages.
I might need to check into this in the future.